### PR TITLE
MOC: Ichor Elixir + support

### PR DESCRIPTION
**Pull request by Northmoc**
A big one!

### CODE REVIEW

File: forge-gui/res/languages/es-ES.properties [hunk 1]
```diff
@@ -2963,6 +2966,6 @@ lblBoon=Boon
 lblExitToWoldMap=Salir al mapa del mundo?
 lblStartArena=¿Quieres ir a la arena?
 lblWouldYouLikeDestroy=¿Le gustaría destruir {0}?
-lblAdventureDescription=�El modo de aventura es donde exploras el paisaje siempre cambiante de Shandalar, y criaturas de duelo para ganar oro, fragmentos, equipos y nuevas tarjetas para convertirte en el mejor y recogerlas a todos!
-lblEmptyDeck=Baraja de cartas vac�a
+lblAdventureDescription=�El modo de aventura es donde exploras el paisaje siempre cambiante de Shandalar, y criaturas de duelo para ganar oro, fragmentos, equipos y nuevas tarjetas para convertirte en el mejor y recogerlas a todos!
+lblEmptyDeck=Baraja de cartas vac�a
```
Comment (tool4ever) [inline forge-gui/res/languages/es-ES.properties:2970]: formatting broke?

Comment (Northmoc) [reply forge-gui/res/languages/es-ES.properties:2970]: Ugh 👀 taking a look

File: forge-game/src/main/java/forge/game/ability/effects/RollPlanarDiceEffect.java [hunk 2]
```diff
@@ -19,17 +18,11 @@ public class RollPlanarDiceEffect extends SpellAbilityEffect {
      */
     @Override
     public void resolve(SpellAbility sa) {
-        boolean countedTowardsCost = !sa.hasParam("NotCountedTowardsCost");
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/ability/effects/RollPlanarDiceEffect.java:22]: don't the rules need this and _Fractured Powerstone_ was missing it?

File: forge-gui/res/languages/es-ES.properties (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/languages/es-ES.properties:2970]: looks like this in Diff to me, but maybe it's my browser?
grafik

**Comment by tool4ever** [comment]
Should be a special action:
https://github.com/Card-Forge/forge/blob/126c708c034a738e265e73407eb6e9fe7625c893/forge-game/src/main/java/forge/game/card/CardFactory.java#L356

File: forge-gui/res/languages/es-ES.properties (hunk 1 above)
Comment (Northmoc) [reply forge-gui/res/languages/es-ES.properties:2970]: No something corrupted it 😞

**Comment by Northmoc** [comment]
> Should be a special action:
> 
> https://github.com/Card-Forge/forge/blob/126c708c034a738e265e73407eb6e9fe7625c893/forge-game/src/main/java/forge/game/card/CardFactory.java#L356

I agree - but I have no idea how to set this up ☹️

File: forge-game/src/main/java/forge/game/ability/effects/RollPlanarDiceEffect.java (hunk 2 above)
Comment (Northmoc) [reply forge-game/src/main/java/forge/game/ability/effects/RollPlanarDiceEffect.java:22]: Fixed it

File: forge-gui/res/languages/zh-CN.properties [hunk 3]
```diff
@@ -2946,5 +2949,5 @@ lblExitToWoldMap=退出世界地图？
 lblStartArena=您想进入竞技场吗？
 lblWouldYouLikeDestroy=您想销毁{0}吗？
 lblAdventureDescription=冒险模式是您探索山达尔（Shandalar）不断变化的景观，以及决斗生物获得黄金，碎片，设备和新卡片，使其成为最好的，并收集所有这些！
-lblEmptyDeck=空牌� � - 空牌� �
-lblNoAvailableSlots=没有可用的槽位�
+lblEmptyDeck=蝛箇�� � - 蝛箇�� �
```
Comment (kevlahnota) [inline forge-gui/res/languages/zh-CN.properties:2952]: lblEmptyDeck=空甲板
lblNoAvailableSlots=没有可用的插槽

**Comment by Hanmac** [comment]
do you want to make #2829 part of this MR too or for the next one?

**Comment by Northmoc** [comment]
Yeah I think this PR is big enough already...

Chaos Ensues stuff will be next

File: forge-game/src/main/java/forge/game/ability/effects/RollPlanarDiceEffect.java [hunk 4]
```diff
@@ -19,17 +19,19 @@ public class RollPlanarDiceEffect extends SpellAbilityEffect {
...
         final Player activator = sa.getActivatingPlayer();
         final Game game = activator.getGame();
 
-        if(countedTowardsCost) {
+        if (game.getActivePlanes() == null) { // not a planechase game, nothing happens
+            return;
+        }
+        if (!sa.hasParam("NotCountedForPDCost")) {
```
Comment (Hanmac) [inline forge-game/src/main/java/forge/game/ability/effects/RollPlanarDiceEffect.java:28]: hm i don't like that Param, can't we just get with a better way which ones should count, and which should not?

it is just for the Static Action, wouldn't it be better to mark the static action instead?

File: forge-game/src/main/java/forge/game/PlanarDice.java [hunk 5]
```diff
@@ -20,36 +21,61 @@ public enum PlanarDice {
...
+            results.remove(roller.getController().choosePDRollToIgnore(results));
+        }
+        PlanarDice res = results.get(0);
 
         PlanarDice trigRes = res;
 
-        if(roller.getGame().getStaticEffects().getGlobalRuleChange(GlobalRuleChange.blankIsChaos)
-                && res == Blank)
-        {
+        if (game.getStaticEffects().getGlobalRuleChange(GlobalRuleChange.blankIsChaos)
```
[the remark this answers is not in the document]
Comment (Northmoc) [reply forge-game/src/main/java/forge/game/PlanarDice.java:63]: Hmm, that might not be too bad, let me give it a go in this one

File: forge-game/src/main/java/forge/game/card/CardFactory.java [hunk 6]
```diff
@@ -341,23 +341,21 @@ private static void buildAbilities(final Card card) {
...
 
-        Trigger planesWalkTrigger = TriggerHandler.parseTrigger(triggerSB.toString(), card, true);
+        Trigger planesWalkTrigger = TriggerHandler.parseTrigger(trigger, card, true);
         planesWalkTrigger.setOverridingAbility(AbilityFactory.getAbility(rolledWalk, card));
         card.addTrigger(planesWalkTrigger);
 
-        StringBuilder saSB = new StringBuilder();
-        saSB.append("ST$ RollPlanarDice | Cost$ X | SorcerySpeed$ True | Activator$ Player | ActivationZone$ Command | ");
-        saSB.append("SpellDescription$ Roll the planar dice. X is equal to the amount of times the planar die has been rolled this turn.");
+        String specialA = "ST$ RollPlanarDice | Cost$ X | SorcerySpeed$ True | Activator$ Player | SpecialAction$ True" +
```
Comment (Hanmac) [inline forge-game/src/main/java/forge/game/card/CardFactory.java:353]: there is a | missing

File: forge-game/src/main/java/forge/game/phase/PhaseHandler.java [hunk 7]
```diff
@@ -1212,11 +1212,11 @@ public final void endTurnByEffect() {
         onPhaseBegin();
     }
 
-    public int getPlanarDiceRolledthisTurn() {
-        return planarDiceRolledthisTurn;
+    public int getPlanarDiceSpecialActionThisTurn() {
```
Comment (kevlahnota) [inline forge-game/src/main/java/forge/game/phase/PhaseHandler.java:1215]: Error:  /home/runner/work/forge/forge/forge-ai/src/main/java/forge/ai/ability/RollPlanarDiceAi.java:[137,47] cannot find symbol
  symbol:   method getPlanarDiceRolledthisTurn()
  location: class forge.game.phase.PhaseHandler

File: forge-game/src/main/java/forge/game/PlanarDice.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (Northmoc) [reply forge-game/src/main/java/forge/game/PlanarDice.java]: Got it!

**Comment by Northmoc** [comment]
this will help with issue #2544 (which almost went stale)

**Review by kevlahnota — APPROVED** [review]

**Pull request merged**